backend/routes/db.py

```diff
@@ -402,7 +418,8 @@ def get_researchers(id: List[str] = Query(...), fields: Union[List[str], None] =
         FROM researcher
         WHERE "multipassId" = ANY(:id)
     """
-    res: List[RowMapping] = sql_query(get_db_connection(), query, {'id': list(id)}, return_with_keys=True)
```

Comment:
## Type 2/3: function call inside function call
I don't know if I saw offending cases of this in the log, but I believe these remain open as well.